### ISSUE TITLE
Improve recognition on A vs 4

### ISSUE THREAD

**Issue by Fed0t** [dharm1k987/Card_Recognizer#1]
Hello, nice setup here, how tensorflow and keras are getting better on prediction? They learn from the webcam? I stop the training for 95% accuracy but i have big problems with A and 4 . The A is predicted as 4.

**Comment by domesticflee** [comment]
no erro no result ,what's this red cross.
